tests/robustness/report/report.go

```diff
@@ -35,51 +30,33 @@ type TestReport struct {
 	Visualize func(lg *zap.Logger, path string) error
 }
 
-func testResultsDirectory(t *testing.T) string {
-	resultsDirectory, ok := os.LookupEnv("RESULTS_DIR")
-	if !ok {
-		resultsDirectory = "/tmp/"
+func (r *TestReport) Report(failed, force, persistResults bool, path string) error {
```

Comment:
Those three variables work identically, making them really function as single one. And the if statement line below just works as a turn off switch. Based on that I think we can just remove all of them and move the condition outside function. Like

```
shouldReport := failed || force || persistResults
if shouldReport {
  t.Report(path)
}
```